tekton-pipeline/task-bump-jenkins.yaml

```diff
@@ -0,0 +1,105 @@
+﻿apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  creationTimestamp: '2020-10-15T08:58:13Z'
+  generation: 17
+  managedFields:
+    - apiVersion: tekton.dev/v1beta1
+      fieldsType: FieldsV1
+      fieldsV1:
+        'f:spec':
+          .: {}
+          'f:params': {}
+          'f:results': {}
+          'f:steps': {}
+          'f:volumes': {}
+      manager: Mozilla
+      operation: Update
+      time: '2020-10-20T09:19:22Z'
+  name: bump-jenkins
+  namespace: art-jenkins
+  resourceVersion: '66517529'
+  selfLink: /apis/tekton.dev/v1beta1/namespaces/art-jenkins/tasks/bump-jenkins
+  uid: 1ba0aa05-35ff-4ea3-8835-612dd692aa33
+spec:
+  params:
+    - name: jenkins-version
+      type: string
+    - name: ocp-branch
+      type: string
+  results:
+    - description: the result of this task
+      name: jobresult
+  steps:
+    - image: docker.io/library/centos
+      name: ''
+      resources: {}
+      script: >
+        #!/usr/bin/env bash
+
+        set -xeuo pipefail
+
+        echo failed > /tekton/results/jobresult
+
+        yum install -q -y wget
```

Comment:
I would like the sole function of these scripts to be relaying job parameters to the real scripts, which are installed in `art-ci-toolkit` and probably python-based for the most part (although TBH this one looks like bash would make sense; still, I want the script in a standalone container file for testing, not in the task definition).

I definitely do not want every task to yum install what it needs every time it runs :) That should be done in the Dockerfile for `art-ci-toolkit` along with pretty much every other common system configuration/installation/setup.